streetwhisperapp.py

```diff
@@ -118,7 +167,30 @@ def questions_ui(diarize_model):
     # Input file
     rprint("[blue]=============================[blue]")
     rprint(f"[bold]Enter the absolute path to the audio file you want to do the \"{process_selected['process_selected']}\" process on:[bold]")
-    input_file = input()
+
+    # Check if audio file path is a valid path within system that CLI is running from
+
+    while True:
+        input_file = input()
+        input_audio_path = input_file.strip()  # remove leading and trailing whitespace
+        # Check 1 for input file: Validate whether path is valid
+        audio_path_last_backslash_index = input_file.rfind("/")
+        audio_name = input_file[audio_path_last_backslash_index + 1:]
+        audio_name = audio_name.strip()  # remove leading and trailing whitespace
+        # TODO: Purposefully left out the "replace <spaces> with <"_"> since that might interfere with the path checking
+        # TODO: Moreover, having spaces in path name still yields an actual valid path (at least for MacOS)
+        input_audio_path = (input_audio_path.strip())[0: input_audio_path.rfind("/") + 1] + audio_name
+        is_valid_audio_path = validate_path(input_audio_path, True)
+        if is_valid_audio_path:
+            break
+        else:
+            print("You entered an invalid audio path. Please try again")
```

Comment:
Maybe include a colon at the end "You entered an invalid audio path. Please try again:" so that user knows that they can enter a new audio path again.